cdptools/databases/cloud_firestore_database.py

```diff
@@ -798,6 +802,27 @@ def search_minutes_items(self, query: str) -> List[Match]:
             data_table="minutes_item"
         )
 
+    def wipe_table(self, table, batch_size):
+        docs = self._root.collection(table).list_documents(batch_size)
+
+        total_del = 0
+        while docs:
+            deleted_count = 0
+            for doc in docs:
+                doc.delete()
+                deleted_count += 1
+                total_del += 1
+
+            # If there could potentially be more documents in the table
+            if deleted_count >= batch_size:
+                docs = self._root.collection(table).list_documents(batch_size)
+            else:
+                log.info("Deleted {} docs from {} table in batches of {} docs".format(total_del, table, batch_size))
+                return
+
+    def _tables(self) -> List[str]:
```

Comment:
I don't _think_ this is going to work.... Since there is an attribute called `_tables` and you are defining this function as `_tables`. I would change to:

```suggestion
    @property
    def tables(self) -> List[str]:
```

Reply:
Thanks this is a great catch! It was still working because Python was defaulting to using the attribute rather than the function, but it wasn't actually using the property getter method. Because it used the attribute it also hid the issue that I forgot the property annotation in the concrete class.